src/nccl_ofi_net.c

```diff
@@ -1199,8 +1199,8 @@ static int set_nic_props_default(int dev_id, struct fi_info *nic_prov,
 	 * objects for devices with multiple ports. So, safely assume port number
 	 * to be always 1.
 	 */
-	props->port = 1;
-	props->maxComms = nic_prov->domain_attr->ep_cnt;
+	props->port_number = 1;
+	props->max_communicators = nic_prov->domain_attr->ep_cnt;
```

Comment:
Were you going to fix this in a separate PR?

Reply:
Yes; this PR was already bordering on being too broad.